data/quarg outfits.txt

```diff
@@ -102,7 +102,7 @@ outfit "Intrusion Countermeasures"
 outfit "Medium Graviton Thruster"
 	category "Engines"
 	"cost" 20000000
-	thumbnail "outfit/medium quarg thruster"
+	thumbnail "outfit/medium quarg thrusters"
```

Comment:
the small & large ones are "[small/large] quarg thruster" so maybe change this back to thruster & rename the "medium quarg thrusters" so singular for consistency?